shadow-cljs.edn

```diff
@@ -17,15 +17,14 @@
                                                  holyjak.fulcro-troubleshooting ; optional, to find mistakes faster
                                                  com.fulcrologic.fulcro.inspect.preload
                                                  com.fulcrologic.fulcro.inspect.dom-picker-preload]
-                                    :http-root   "resources/public/" ;; Must be set to infer default purge targets.
-                                    :http-port   3000}
```

Comment:
@zacjones93 you didnt need to add these values... they created 2 more builds for shadow cljs to track. We are using port 8000

Reply:
ah, good to know